lib/quantum/job_broadcaster.ex

```diff
@@ -52,8 +54,8 @@ defmodule Quantum.JobBroadcaster do
       end
 
     state = %{
-      jobs: Enum.into(jobs, %{}, fn %{name: name} = job -> {name, job} end),
-      buffer: for(%{state: :active} = job <- jobs, do: {:add, job}),
+      jobs: Enum.into(effective_jobs, %{}, fn %{name: name} = job -> {name, job} end),
+      buffer: for(%{state: :active} = job <- effective_jobs, do: {:add, job}),
       storage: storage,
       scheduler: scheduler
```

Comment:
Just realized that the existing code was rebinding the `jobs` variable, so...`effective_jobs` variable is basically not needed.
I could probably stick with the `jobs` variable.
I mean this part could be reverted (what existed in the master repo `storage` branch is just fine).

Reply:
It's good like this 👍